engine/src/test/java/org/hibernate/validator/test/internal/constraintvalidators/hv/URLValidatorTest.java

```diff
@@ -363,22 +364,16 @@ private void assertValidUrls(ConstraintValidator<URL, CharSequence> validator) {
 			assertFalse( validator.isValid( "http:///a", null ) );
 			assertFalse( validator.isValid( "http:// shouldfail.com", null ) );
 			assertFalse( validator.isValid( "http://foo.bar/foo(bar)baz quux", null ) );
-//			assertFalse( validator.isValid( "http://-error-.invalid/", null ) );
-//			assertFalse( validator.isValid( "http://a.b--c.de/", null ) );
-//			assertFalse( validator.isValid( "http://-a.b.co", null ) );
-//			assertFalse( validator.isValid( "http://a.b-.co", null ) );
-//			assertFalse( validator.isValid( "http://0.0.0.0", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.0", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.255", null ) );
-//			assertFalse( validator.isValid( "http://224.1.1.1", null ) );
-//			assertFalse( validator.isValid( "http://1.1.1.1.1", null ) );
+			assertFalse( validator.isValid( "http://-error-.invalid/", null ) );
+			assertFalse( validator.isValid( "http://a.b--c.de/", null ) );
+			assertFalse( validator.isValid( "http://-a.b.co", null ) );
+			assertFalse( validator.isValid( "http://a.b-.co", null ) );
 //			assertFalse( validator.isValid( "http://123.123.123", null ) );
 //			assertFalse( validator.isValid( "http://3628126748", null ) );
 			assertFalse( validator.isValid( "http://.www.foo.bar/", null ) );
 			assertFalse( validator.isValid( "http://www.foo.bar./", null ) );
 			assertFalse( validator.isValid( "http://.www.foo.bar./", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.1", null ) );
-//			assertFalse( validator.isValid( "http://10.1.1.254", null ) );
+			assertFalse( validator.isValid( "http://2001:0db8:0a0b:12f0:0000:0000:0000:0001", null ) );
```

Comment:
In the regexp, it looks like you put `[]` around the IPv6 IP address but here you just put the address without any wrapper? Did I miss something?